[2022-05-22 22:13:15,513] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: book_db.check_api 2022-05-22T22:13:10.207470+00:00 [queued]>
[2022-05-22 22:13:15,538] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: book_db.check_api 2022-05-22T22:13:10.207470+00:00 [queued]>
[2022-05-22 22:13:15,539] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 22:13:15,540] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2022-05-22 22:13:15,541] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 22:13:15,595] {taskinstance.py:1107} INFO - Executing <Task(HttpSensor): check_api> on 2022-05-22T22:13:10.207470+00:00
[2022-05-22 22:13:15,599] {standard_task_runner.py:52} INFO - Started process 48 to run task
[2022-05-22 22:13:15,609] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'book_db', 'check_api', '2022-05-22T22:13:10.207470+00:00', '--job-id', '9', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/books_db.py', '--cfg-path', '/tmp/tmpo10hzpqp', '--error-file', '/tmp/tmp0qvh9x1n']
[2022-05-22 22:13:15,611] {standard_task_runner.py:77} INFO - Job 9: Subtask check_api
[2022-05-22 22:13:15,708] {logging_mixin.py:104} INFO - Running <TaskInstance: book_db.check_api 2022-05-22T22:13:10.207470+00:00 [running]> on host 0f09442c242c
[2022-05-22 22:13:15,895] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=book_db
AIRFLOW_CTX_TASK_ID=check_api
AIRFLOW_CTX_EXECUTION_DATE=2022-05-22T22:13:10.207470+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-22T22:13:10.207470+00:00
[2022-05-22 22:13:15,897] {http.py:101} INFO - Poking: new
[2022-05-22 22:13:15,911] {base.py:69} INFO - Using connection to: id: book_api. Host: https://api.itbook.store/1.0/, Port: None, Schema: , Login: , Password: None, extra: {}
[2022-05-22 22:13:15,916] {http.py:140} INFO - Sending 'GET' to url: https://api.itbook.store/1.0/new
[2022-05-22 22:13:16,411] {base.py:248} INFO - Success criteria met. Exiting.
[2022-05-22 22:13:16,438] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=book_db, task_id=check_api, execution_date=20220522T221310, start_date=20220522T221315, end_date=20220522T221316
[2022-05-22 22:13:16,731] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-05-22 22:13:16,791] {local_task_job.py:151} INFO - Task exited with return code 0
